source/index.js

```diff
@@ -48,6 +48,11 @@ class Storyblok {
       headers: headers,
       proxy: (config.proxy || false)
     })
+    if (config.responseInterceptor) {
+      this.client.interceptors.response.use((res) => {
+        return config.responseInterceptor(res);
```

Comment:
Small coding style issue. Please remove the semicolon ";" as in the rest of the project

Reply:
Sure. I'll do!

Reply:
Done!